packages/zod/src/v4/core/tests/locales/he.test.ts

```diff
@@ -0,0 +1,370 @@
+        const result = schema.safeParse(input);
+        expect(result.success).toBe(false);
+        if (!result.success) {
+          expect(result.error.issues[0].message).toContain("צריך");
+        }
+      }
+    });
+  });
+
+  describe("invalid_value with enum", () => {
+    test("single value", () => {
+      const schema = z.enum(["a"]);
+      const result = schema.safeParse("b");
+      expect(result.success).toBe(false);
+      if (!result.success) {
+        expect(result.error.issues[0].message).toBe('קלט לא תקין: צריך "a"');
+      }
+    });
+
+    test("multiple values", () => {
+      const schema = z.enum(["a", "b", "c"]);
+      const result = schema.safeParse("d");
+      expect(result.success).toBe(false);
+      if (!result.success) {
+        expect(result.error.issues[0].message).toBe('קלט לא תקין: צריך אחת מהאפשרויות "a" | "b" | "c"');
+      }
+    });
+  });
+
+  describe("other error types", () => {
+    test("not_multiple_of", () => {
+      const schema = z.number().multipleOf(3);
+      const result = schema.safeParse(10);
+      expect(result.success).toBe(false);
+      if (!result.success) {
+        expect(result.error.issues[0].message).toBe("מספר לא תקין: חייב להיות מכפלה של 3");
+      }
+    });
+
+    test("unrecognized_keys - single key", () => {
+      const schema = z.object({ a: z.string() }).strict();
+      const result = schema.safeParse({ a: "test", b: "extra" });
+      expect(result.success).toBe(false);
+      if (!result.success) {
+        expect(result.error.issues[0].message).toBe('מפתח לא מזוהה: "b"');
+      }
+    });
+
+    test("unrecognized_keys - multiple keys", () => {
+      const schema = z.object({ a: z.string() }).strict();
+      const result = schema.safeParse({ a: "test", b: "extra", c: "more" });
+      expect(result.success).toBe(false);
+      if (!result.success) {
+        expect(result.error.issues[0].message).toBe('מפתחות לא מזוהים: "b", "c"');
+      }
+    });
+
+    test("invalid_union", () => {
+      const schema = z.union([z.string(), z.number()]);
+      const result = schema.safeParse(true);
+      expect(result.success).toBe(false);
+      if (!result.success) {
+        expect(result.error.issues[0].message).toBe("קלט לא תקין");
+      }
+    });
+
+    test("invalid_key in object", () => {
+      const schema = z.record(z.number(), z.string());
+      const result = schema.safeParse({ notANumber: "value" });
+      expect(result.success).toBe(false);
+      if (!result.success) {
+        expect(result.error.issues[0].message).toContain("מפתח לא תקין");
+      }
+    });
+  });
+
+  describe("invalid_format with string checks", () => {
+    test("startsWith", () => {
+      const schema = z.string().startsWith("hello");
+      const result = schema.safeParse("world");
+      expect(result.success).toBe(false);
+      if (!result.success) {
+        expect(result.error.issues[0].message).toBe('המחרוזת חייבת להתחיל ב"hello"');
```

Comment:
```suggestion
        expect(result.error.issues[0].message).toBe('המחרוזת חייבת להתחיל ב "hello"');
```

i think this is better, wdyt?
also others